zebra-consensus/src/parameters/subsidy.rs

```diff
@@ -0,0 +1,94 @@
+//! Constants for Block Subsidy, Funding Streams, and Founders’ Reward
+
+use zebra_chain::parameters::Network;
+
+/// Whatever
+pub struct Params(pub Network);
+
+impl Params {
+    /// COIN
+    pub const COIN: u32 = 100000000;
+
+    /// SlowStartInterval
+    pub const SLOW_START_INTERVAL: u32 = 20000;
+
+    /// MaxBlockSubsidy
+    pub const MAX_BLOCK_SUBSIDY: u32 = (12.5 * (Self::COIN as f32)) as u32;
+
+    /// PreBlossomPoWTargetSpacing
+    pub const PRE_BLOSSOM_POW_TARGET_SPACING: u32 = 150;
+
+    /// PostBlossomPoWTargetSpacing
+    pub const POST_BLOSSOM_POW_TARGET_SPACING: u32 = 75;
+
+    /// BLOSSOM_POW_TARGET_SPACING_RATIO
+    pub const BLOSSOM_POW_TARGET_SPACING_RATIO: u32 =
+        Self::PRE_BLOSSOM_POW_TARGET_SPACING / Self::POST_BLOSSOM_POW_TARGET_SPACING;
+
+    /// PreBlossomHalvingInterval
+    pub const PRE_BLOSSOM_HALVING_INTERVAL: u32 = 840000;
+
+    /// POST_BLOSSOM_HALVING_INTERVAL
+    pub const POST_BLOSSOM_HALVING_INTERVAL: u32 =
+        Self::PRE_BLOSSOM_HALVING_INTERVAL * Self::BLOSSOM_POW_TARGET_SPACING_RATIO;
+
+    /// MAX_MONEY
+    pub const MAX_MONEY: u64 = 21000000 * Self::COIN as u64;
```

Comment:
This constant already exists:

```suggestion
    use zebra_chain::amount::MAX_MONEY;
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/1051/commits/fa7d722886c9dbd2f65a48c348158c23ec842fa8